runtests.sh

```diff
@@ -778,6 +870,12 @@ alltests() {
   echo ""
 }
 
+export_testvars
```

Comment:
Otherwise things seem to work - remove those lines and merge if you want. Or tell me and I'll do it.